src/BloomBrowserUI/lib/long-press/jquery.longpress.js

```diff
@@ -164,13 +170,13 @@
         }
     }
     function showPopup(chars) {
-        popup.empty();
+        popup.find('ul').empty();
```

Comment:
See line 116/117 where the definition/structure of popup was changed to make room for instructions.